### DISCUSSION TITLE
Any commandlines we can use to control installs on Windows?

### DISCUSSION THREAD

**Discussion by byjrack** [usebruno/bruno#1180] [Q&A]
the --silent isn't quite very silent and the runafterfinish is enabled.  Just makes it more complicated to handle this for guiding people on use.  Tried tossing a /? on the cmdline and nothing and electron-builder I don't think puts any of the internal NSIS control logic on params either.  Am I missing a doc?

**Comment by helloanoop** [comment]
> Any commandlines we can use to control installs on Windows

@byjrack Are you talking about using `winget` or `scoop` to install bruno on windows ?

> the --silent isn't quite very silent and the runafterfinish is enabled. 

@byjrack Can you expand more. Any chance you could share a screenshot ?

**Reply by byjrack** [reply]
In this case I am using winget and took a peek at the manifest and it just had the -s, --silent tacked on. 

Looks like [scoop uses the portable](https://github.com/ScoopInstaller/Extras/blob/master/bucket/bruno.json) installer so yeah that would avoid the popup since its just an unzip I think.

**Reply by byjrack** [reply]
[Winget supports portable apps](https://github.com/microsoft/winget-cli/blob/master/doc/specs/%23182%20-%20Support%20for%20installation%20of%20portable%20standalone%20apps.md) as well, but needs some tweaks of the manifest.  Folks can adjust which type using the --installer-type if there was a need to keep the EXE as the preferred/default.

**Reply by byjrack** [reply]
And screenshot would just be the Bruno UI which I know you have seen plenty of for the "runafterfinish" param used in the NSIS installer.  --silent just shows the installer progress UI (vs being really silent) and then launches bruno after install because electorn-builder i believe [has that flag set](https://www.electron.build/configuration/nsis.html).
